lib/Model/DoctrineUser.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Model;
+
+use Doctrine\SkeletonMapper\Hydrator\HydratableInterface;
+use Doctrine\SkeletonMapper\Mapping\ClassMetadataInterface;
+use Doctrine\SkeletonMapper\Mapping\LoadMetadataInterface;
+use Doctrine\SkeletonMapper\ObjectManagerInterface;
+
+class DoctrineUser implements HydratableInterface, LoadMetadataInterface
+{
+    /** @var string */
+    private $name;
+
+    /** @var string */
+    private $url;
+
+    public static function loadMetadata(ClassMetadataInterface $metadata) : void
+    {
+        $metadata->setIdentifier(['name']);
+    }
+
+    /**
+     * @param mixed[] $doctrineUser
+     */
+    public function hydrate(array $doctrineUser, ObjectManagerInterface $objectManager) : void
```

Comment:
Types for the properties should be handled in this method or we can get an invalid object state.